docs/deployment.md

```diff
@@ -54,18 +183,160 @@ docker run \
   -p6832:6832/udp \
   -p5778:5778/tcp \
   jaegertracing/jaeger-agent \
-  /go/bin/agent-linux --collector.host-port=jaeger-collector.jaeger-infra.svc:14267
+  /go/bin/agent-linux
+--collector.host-port=jaeger-collector.jaeger-infra.svc:14267
 ```
 
-In the future we will support different service discovery systems to dynamically load balance
-across several collectors ([issue 213](https://github.com/uber/jaeger/issues/213)).
+In the future we will support different service discovery systems to
+dynamically load balance
+across several collectors ([issue
+213](https://github.com/uber/jaeger/issues/213)).
```

Comment:
what is the purpose of these formatting changes? they seem to make this diff twice as big as it needs to be

Reply:
Fixed that ,thanks :)